src/main/java/com/afwsamples/testdpc/DevicePolicyManagerGatewayImpl.java

```diff
@@ -636,7 +638,11 @@ public void setPreferentialNetworkServiceEnabled(
   public boolean isPreferentialNetworkServiceEnabled() {
     Util.requireAndroidS();
 
-    return mDevicePolicyManager.isPreferentialNetworkServiceEnabled();
+    if (isAtLeastT()) {
```

Comment:
There is undoubtably a more thorough & better fix but this allows the activity to launch, at least.